sync.sh

```diff
@@ -51,7 +51,7 @@ $working_tree_root/init-tools.sh
 
 if [ "$sync_src" == true ]; then
     echo "Fetching git database from remote repos..."
-    git fetch --all -p -v &>> $sync_log
```

Comment:
Me neither. 
Yes, I tested it in Ubuntu and OS X

@naamunds why do we have `&` there?